CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## v0.30.1 [2023-07-24]
+
+_Bug fixes_
+
+- Fixed the `labels` column to correctly return data instead of throwing an error. ([#303](https://github.com/turbot/steampipe-plugin-github/pull/303))
```

Comment:
@madhushreeray30 could you please mention the tables that this column belongs to in the release note as well?